Headers/AppKit/NSView.h

```diff
@@ -590,6 +615,19 @@ PACKAGE_SCOPE
 - (NSAttributedString *)pageHeader;
 #endif
 
+#if OS_API_VERSION(MAC_OS_X_VERSION_10_6, GS_API_LATEST)
+#if GS_HAS_DECLARED_PROPERTIES
+@property NSViewLayerContentsPlacement layerContentsPlacement;
```

Comment:
I think you want to tell the compiler that these properties are implemented as custom, non-synthesized methods, i.e. `nonatomic`, right? (I recall OS X's clang printing that as a warning.)